dynawo/sources/Models/Modelica/Dynawo/Electrical/Controls/Voltage/SecondaryVoltageControlSimp.mo

```diff
@@ -0,0 +1,83 @@
+within Dynawo.Electrical.Controls.Voltage;
+
+/*
+* Copyright (c) 2022, RTE (http://www.rte-france.com)
+* See AUTHORS.txt
+* All rights reserved.
+* This Source Code Form is subject to the terms of the Mozilla Public
+* License, v. 2.0. If a copy of the MPL was not distributed with this
+* file, you can obtain one at http://mozilla.org/MPL/2.0/.
+* SPDX-License-Identifier: MPL-2.0
+*
+* This file is part of Dynawo, an hybrid C++/Modelica open source suite of simulation tools for power systems.
+*/
+
+model SecondaryVoltageControlSimp "Model for simplified secondary voltage control"
```

Comment:
I find it strange to suffix by "Simp" here (ie SecondaryVoltageControlSimp) and not to do it with the RPCL. I would even be in favor of keeping SecondaryVoltageControl

Reply:
I can remove it, or use Simplified also for the RPCL, just like we did a simplified folder for the voltage regulators and governors.
Anyway, I will need to use a different name than just SecondaryVoltageControl in the preassembled as it already exists in dynawo-rte for DynaWaltz

Reply:
Good point for the preassembled, I haven't thought about it.
Yes maybe having a Simplified folder could be the best idea, and be consistent with the organization in other parts